src/libs/Navigation/drawerGoBack/index.android.js

```diff
@@ -0,0 +1,19 @@
+import {Keyboard} from 'react-native';
```

Comment:
this won't be needed for iOS?

Reply:
yup because on IOS everything is handle by navigation itself, the issue only for android